### PR TITLE
django-admin: add page

### PR DESCRIPTION
**Pull request by shrimp509**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/django-admin.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# django-admin
+
+> A server-side MVT framework written in Python.
+> More information: <https://www.djangoproject.com>.
+
+- Create a new django project:
+
+`django-admin startproject {{project_name}}`
+
+- Create a new app for current project:
```
Comment (zdroid) [inline pages/common/django-admin.md:10]:
```suggestion
- Create a new app for the current project:
```

File: pages/common/django-admin.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`django-admin startapp {{app_name}}`
+
+- Check current version of django:
```
Comment (zdroid) [inline pages/common/django-admin.md:14]:
```suggestion
- Check the current version of Django:
```

File: pages/common/django-admin.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`django-admin --version`
+
+- Check specific help page of django:
```
Comment (zdroid) [inline pages/common/django-admin.md:18]:
```suggestion
- Check a specific help page of Django:
```

File: pages/common/django-admin.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# django-admin
+
+> A server-side MVT framework written in Python.
+> More information: <https://www.djangoproject.com>.
```
Comment (zdroid) [inline pages/common/django-admin.md:4]: Is there a more appropriate link and description, related to django-admin CLI tool specifically?

**Comment by zdroid** [comment]
Just a few minor suggestions. Thanks for the contribution!

File: pages/common/django-admin.md (hunk 4 above)
Comment (shrimp509) [reply pages/common/django-admin.md:4]: Maybe this link is more specific to django-admin CLI tool.
https://docs.djangoproject.com/en/3.0/ref/django-admin/

Comment (zdroid) [reply pages/common/django-admin.md:4]: Yup, that's nice.
```suggestion
> More information: <https://docs.djangoproject.com/en/3.0/ref/django-admin/>.
```

File: pages/common/django-admin.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# django-admin
+
+> A server-side MVT framework written in Python.
```
Comment (zdroid) [inline pages/common/django-admin.md:3]:
```suggestion
> Django’s command-line utility for administrative tasks. 
```

**Comment by shrimp509** [comment]
> The [build](https://travis-ci.org/tldr-pages/tldr/builds/691272512) for this PR failed with the following error(s):
> 
> ```
> pages/common/django-admin.md:3: TLDR004 Command descriptions should end in a period
> ```
> 
> Please fix the error(s) and push again.

Any misunderstand here? 
line 3 is end in a period. What's wrong?

**Comment by shrimp509** [comment]
Never mind. I found it...

File: pages/common/django-admin.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# django-admin
+
+> Django’s command-line utility for administrative tasks.
```
Comment (schneiderl) [inline pages/common/django-admin.md:3]: It being a command-line tool is already implicit. Maybe we can change it to
```suggestion
> Django’s utility for administrative tasks.
```

**Comment by shrimp509** [comment]
Thanks for your help. You made this look great.

File: pages/common/django-admin.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+- Create a new app for the current project:
+
+`django-admin startapp {{app_name}}`
+
+- Check the current version of Django:
+
+`django-admin --version`
+
+- Check a specific help page of Django:
```
Comment (andrik) [inline pages/common/django-admin.md:18]:
```suggestion
- Display more information of the given command:
```

Comment (zdroid) [reply pages/common/django-admin.md:18]:
```suggestion
- Display more information for the given command:
```
how about this?

Comment (shrimp509) [reply pages/common/django-admin.md:18]: Great. I'll go along with @zdroid's suggestion. Thanks @andrik, this is much better.

**Pull request merged**